PROVISIONING-METAL.md

```diff
@@ -106,10 +107,46 @@ Please keep in mind that when using static addresses, DNS information must be su
   * `via` (IP address): Gateway IP address.  If no gateway is provided, a scope of `link` is assumed.
   * `route-metric` (integer): Relative route priority.
 
-Example `net.toml` with comments:
+Version `3` adds in support for bonding and vlan tagging. 
+The support is limited to mode `1` (`active-backup`) for [bonding](https://www.kernel.org/doc/Documentation/networking/bonding.txt).
+Future support may include other bonding options - pull requests are welcome! 
+Version `3` also adds the concept of network devices vs network interfaces.
+The default type of device is an interface and the syntax is the same as previous versions.
+The name of an interface must match an existing interface on the system such as `eno1` or `enp0s16`.
+For virtual network devices, a `kind` is required.
+If no `kind` is specified, it is assumed to be an interface.
+Currently, `bond` and `vlan` are the two supported `kind`s.
+Virtual network devices are created, and therefore a name has to be chosen.
+Names for virtual network devices must conform to [kernel naming requirements](https://elixir.bootlin.com/linux/v5.10.102/source/include/uapi/linux/if.h#L33).
```

Comment:
This link isn't particularly helpful - perhaps we should enumerate the short list of requirements here?

Reply:
I'll add it to be more explicit instead.